### ISSUE TITLE
Issue with postmarketos-demos

### ISSUE THREAD

**Issue by drebrez** [postmarketOS/pmbootstrap#533]
The latest version of postmarketos-demos fails with the following error:
```
localhost:~# postmarketos-demos
process 1169: D-Bus library appears to be incorrectly set up; failed to read machine uuid: Failed to open "/etc/machine-id": No such file or directory
See the manual page for dbus-uuidgen to correct this issue.
  D-Bus not compiled with backtrace support so unable to print a backtrace
Aborted
```

**Comment by ollieparanoid** [comment]
*Some context: the machine ID is generated by systemd, in case it does not exist. Here is some [more  information](https://www.freedesktop.org/software/systemd/man/machine-id.html) about it. Alpine works without systemd, but we need to run eudev for a lot of programs (which is a forked version  of udev, without systemd, because udev was merged into systemd).*

Looks like eudev requires that `/etc/machine-id` to be present, too. In OpenRC/Alpine, that file actually gets generated in the dbus init script, see `/etc/init.d/dbus`:
```shell
# ...
start() {
        ebegin "Starting D-BUS system messagebus"
        /usr/bin/dbus-uuidgen --ensure=/etc/machine-id

        # We need to test if /var/run/dbus exists, since script will fail if it does not
        [ ! -e /var/run/dbus ] && mkdir /var/run/dbus

        start-stop-daemon --start --pidfile /var/run/dbus.pid --exec /usr/bin/dbus-daemon -- --system
        eend $?
}
# ...
```

So my conclusion is, that the error message will go away, when you start dbus once with `service dbus start`, or permanently at boot with `rc-update add dbus default`.


Could you test whether the error goes away, when you manually start the dbus service?

Also if it does, I think the proper fix here would be to enable the dbus service in `postmarketos-ui-weston` in the post-install script (similar to how we enable services in `postmarketos-base`). Could you make a PR for that, in case everything's working?

*On a side note, I've thought about enabling it everywhere by default. But for the console interface for example, one will not need d-bus at all. So I would rather enable it in the UI packages, that actually need it.*

Thank you for reporting!

**Comment by drebrez** [comment]
@ollieparanoid I have to check again, but yesterday I tried to run `/usr/bin/dbus-uuidgen` and `service dbus start` but it was like the dbus wasn't present. I will check again later.

**Comment by drebrez** [comment]
@ollieparanoid I tested it by zapping all the packages and rebuilding everything, the `dbus` package is not installed anymore, I think that the reason is that they removed the `dbus` dependency from the `wpa_supplicant` package (https://git.alpinelinux.org/cgit/aports/commit/?id=49794390171fe37829d0f4f162823b43917116dc) and no other package that we use have it as a dependency.
I tried to just install `dbus` and everything works as expected, even without starting the service and without the `/etc/machine-id` file.
My suggested solution is to simply add the `dbus` package as a dependency of `postmarketos-demos`, because everything else works without dbus (weston and their sample programs), what you think?
